build/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh

```diff
@@ -16,6 +16,11 @@ DEST="$2"
 BAZEL_BIN=$(bazel info bazel-bin --config ci)
 
 bazel run @go_sdk//:bin/go get "github.com/cockroachdb/pebble@$PEBBLE_BRANCH"
+
+# Remove the patch, which doesn't work with older versions. The "unpatched"
+# config assumes the invariants tag is set.
+rm -f build/patches/com_github_cockroachdb_pebble.patch
```

Comment:
Have you tried this? I'm pretty sure the build will fail because it's looking for this file but it doesn't exist.

Making the file contents empty may work, I think if you apply an empty diff then nothing happens?